docs/maintainers/issue_backlog.md

```diff
@@ -0,0 +1,176 @@
+# SIG CLI Issue Backlog
+
+Grooming work for new and existing contributors
+
+- Link: https://goo.gl/YEq33R
+- Author: @pwittrock
+- Last updated: 10/25/17
+
+
+## Background
+
+A goal of SIG CLI is to keep the SIG open to contributions from anyone willing to do the work to get involved. 
+While many folks have shown interest in contributing, we have struggled to keep most folks engaged. 
+Kubernetes recently conducted a survey of new contributors, and major themes were that:
+
+- Contributors don’t know where to start
+- There are too many details to learn before contributing
+- Communication is hard / everyone is too busy to help
+- Hard to get reviewers on PRs
+
+These challenges can be reduced through grooming a work backlog carefully and intentionally.  This address the issues by:
+
+- Providing a backlog for contributors to browse and pull work from.
+- Scoping work that can be done with minimal experience so folks can pick up and work on it.  Mark issues as good
+  first time issues if they can be done by someone with no experience.
+- Reducing the need for constant communication by having the work be well defined and clearly scoped in the issues
+  themselves.
+- Ensure each issue has a stake holder that is committed to seeing it through.  This individual will be motivated to
+  make sure it is reviewed.
+
+## Contribution lifecycle
+
+1. Issue is created with description and labels
+  - Made sure the issue meets guidelines for a good issue
+  - Issue must have a stakeholder invested in the work being completed
```

Comment:
Any ideas on how the stakeholder / person-who-is-willing-to-review-PR identified?